cmd/genji/shell/command.go

```diff
@@ -136,3 +142,193 @@ func displaySuggestions(in string) error {
 	fmt.Println()
 	return nil
 }
+
+// dumpTable displays the content of the given table as SQL statements.
+func dumpTable(tx *genji.Tx, t *database.Table) error {
```

Comment:
We should receive an `io.Writer` and write to it, instead of writing directly to stdout:
- this will make this function almost ready to be used as a separate command
- we will be able to write tests for this, which we don't have at the moment

Reply:
I didn't dare to do it.